SimG4CMS/Calo/interface/HGCMouseBite.h

```diff
@@ -0,0 +1,26 @@
+#ifndef SimG4CMS_HGCMouseBite_h
+#define SimG4CMS_HGCMouseBite_h
+
+#include "Geometry/HGCalCommonData/interface/HGCalDDDConstants.h"
+#include "G4ThreeVector.hh"
+
+#include <vector>
+
+class HGCMouseBite {
+
+public:    
+
+  HGCMouseBite(const HGCalDDDConstants& hgc, const std::vector<double>& angle,
+	       double maxLength, bool waferRotate); 
+  virtual ~HGCMouseBite();
```

Comment:
When designing a class interface, ask yourself the following three questions:

* Do you need to anchor the vtbl (doesn’t apply to class templates)?
* Does proper destruction of data members require additional declarations or functionality that is not available in the class interface? Does the destruction need to be done consistently with construction (e.g., using the same runtime)?
* Do you need to define a stable interface and chances are that later you may have to add some functionality to the destructor?
If the answers to all these questions are “No”, then let the compiler provide the default implementation of the destructor.